src/Heading.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <header className="App-header">
+      <img
+        src="https://i.pinimg.com/736x/10/ff/aa/10ffaadab6bc3c4c1dd4a3e44bf6d5ad.jpg"
+        alt=""
+        width="40"
+        height="25"
+      />
+      CYF Hotel
+    </header>
+  );
+};
```

Comment:
this is good, you've extracted out the header code into it's own component. This means the component is now reusable can be used throughout the app (imagine you wanted lots of different headers, you could use the same `Header` component and just pass in different data)